generatedocs/data/actions/unpublishbookmark/examples.md

```diff
@@ -0,0 +1,27 @@
+### Example
+
+Unpublish bookmark previously created in the script with an id `bookmark1`.
+
+```json
+{
+    "label" : "Unpublish bookmark 1",
+    "action": "unpublishbookmark",
+    "disabled" : false,
+    "settings" : {
+        "id" : "bookmark1"
+    }
+}
+```
+
+Unpublish bookmark with title "bookmark of testuser", where `testuser` is the username of the simulated user. 
```

Comment:
Maybe change to "Unpublish the bookmark with the `title` "bookmark of testuser", where "testuser" is the username of the simulated user."?